napari/_qt/qt_dims.py

```diff
@@ -328,3 +329,128 @@ def focus_down(self):
         else:
             index = (displayed.index(self.last_used) - 1) % len(displayed)
             self.last_used = displayed[index]
+
+    def play_dim(self, axis: int = 0, fps: float = 10, range=None):
+        """
+        Animate (play) axis
+
+        Parameters
+        ----------
+        axis: int
+            Index of axis to play
+        fps: float
```

Comment:
What do you think about if `fps < 0` can we move through the axis in reverse, and if `fps=0` we call `stop`?